example/src/main/java/org/jadice/gwt/spring/demo/server/TimeServiceImpl.java

```diff
@@ -6,7 +6,8 @@
 
 import org.jadice.gwt.spring.demo.client.TimeService;
 
-import com.google.gwt.user.server.rpc.RemoteServiceServlet;
+import com.google.gwt.user.server.rpc.jakarta.RemoteServiceServlet;
+

```

Comment:
we should have probably already fixed that in the previous version, but latest for 2.12 this import is mandatory and the javax won't work anymore.